agent/tests/server_interfaces/test_olicense.py

```diff
@@ -20,7 +21,13 @@ def test_get_olicense_commands_list(olicense_server: OLicenseLicenseServer):
     Do the commands for invoking the license server have the correct data?
     """
     commands_list = olicense_server.get_commands_list()
-    assert commands_list == [f"{settings.OLIXTOOL_PATH} -sv 127.0.0.1:2345"]
+    assert commands_list == [
+        [
+            f"{settings.OLIXTOOL_PATH}",
+            "-sv",
+            "127.0.0.1:2345",
+        ]
+    ]
```

Comment:
so, I'm confuses why the return for the `get_commands_list` is a list of lists. I mean, each license server has only one command (I noticed by the tests), right? If so, why to return a list of lists?

Reply:
was it designed in case there are more than one license server per license?

Reply:
Each license configuration can have `n` license servers, so this function needs to generate the command for each entry.

Reply:
Some license servers have redundant mirrors, in case one of them fails we use the next one and so on.